Config: adjust a paragraph in 'latex_table_style' documentation (#12994)
sphinx-doc/sphinx: doc/usage/configuration.rst

         Please update your project to use the
         :ref:`latex table color configuration <tablecolors>` keys instead.

   To customize the styles on an individual table, either set the ``:class:``
   option on the enclosing table directive - if there is one - or insert a
   :ref:`rst-class <rstclass>` directive before the table otherwise

>    (cf. :ref:`table-directives`).

Reviewer: Perhaps?:
```suggestion
   To customise the styles for a table,
   use the ``:class:`` option if the table is defined using a directive,
   or otherwise insert a :ref:`rst-class <rstclass>` directive before the table
   (cf. :ref:`table-directives`).
```

Author: Mostly looks good - is the word `defined` suitable here, though?  (as a reader, it makes sense, but I'm not sure it's consistent with the existing documentation)

Reviewer: Happy to go with what you think best.

Author: Let's go with that - I think I was being overly-fussy about the word `defined`.

>    To customize the styles on an individual table, either set the ``:class:``
>    option when the table's contents are documented inside a table directive,
>    or, otherwise, add a preceding :ref:`rst-class <rstclass>` directive
>    (cf. :ref:`table-directives`).

Author: ```suggestion
   To customize the styles on an individual table, either set the ``:class:``
   option on the enclosing table directive - if there is one - or insert a
   :ref:`rst-class <rstclass>` directive before the table otherwise
   (cf. :ref:`table-directives`).
```
